prusti-encoder/src/encoders/mir_pure.rs

```diff
@@ -586,7 +622,11 @@ impl<'vir, 'enc> Encoder<'vir, 'enc>
     ) -> Update<'vir> {
         let mut update = Update::new();
         match &stmt.kind {
-            mir::StatementKind::StorageLive(..)
+            mir::StatementKind::StorageLive(local) => {
```

Comment:
I don't think what i am doing here is correct but `mod_locals` only picks up locals that were defined before the branch and in the particular example here StorageLive touched that variable before. What should i be doing here?